src/tests/unit/tests/electron/platform/android/rule-information-provider.test.ts

```diff
@@ -63,14 +60,14 @@ describe('RuleInformationProvider', () => {
 
     test('getRuleInformation returns correct data for ColorContrast rule', () => {
         const testRuleId: string = 'ColorContrast';
-        const ruleResult: RuleResultsData = buildColorContrastRuleResultObject('FAIL', 2.798498811425733, 'ff979797', 'fffafafa');
+        const ruleResult: RuleResultsData = buildColorContrastRuleResultObject('FAIL', 2.798498811425733, 'ff979797', 'fffafafa', 'High');
         const unifiedResolution: UnifiedResolution = validateUnifiedResolution(testRuleId, ruleResult);
         expect(unifiedResolution).toMatchSnapshot();
     });
 
     test('getRuleInformation handles no foreground/background color values', () => {
         const testRuleId: string = 'ColorContrast';
-        const ruleResult: RuleResultsData = buildColorContrastRuleResultObject('FAIL', 2.798498811425733, null, null);
+        const ruleResult: RuleResultsData = buildColorContrastRuleResultObject('FAIL', null, null, null, 'None');
```

Comment:
The ratio here is based on the following result that was received from the service--notice that the ratio is omitted:
```
    {
      "axeViewId": "-1716372713",
      "props": {
        "Background Color": null,
        "Visible Text": "Display",
        "Confidence in Color Detection": "None",
        "className": "android.widget.TextView",
        "Foreground Color": null
      },
      "ruleId": "ColorContrast",
      "ruleSummary": "Text adequately contrasts with its background.",
      "status": "PASS"
    },
```